pkgs/misc/arm-trusted-firmware/default.nix

```diff
@@ -1,134 +1,17 @@
-          # this is a rebased version of https://gitlab.com/vicencb/kevinboot/-/blob/master/atf.patch
-          ./remove-hdcp-blob.patch
-        ];
-
-        postPatch = lib.optionalString deleteHDCPBlobBeforeBuild ''
-          rm plat/rockchip/rk3399/drivers/dp/hdcp.bin
-        '';
-
-        depsBuildBuild = [ buildPackages.stdenv.cc ];
-
-        nativeBuildInputs = [
-          pkgsCross.arm-embedded.stdenv.cc # For Cortex-M0 firmware in RK3399
-          openssl # For fiptool
-        ];
-
-        # Make the new toolchain guessing (from 2.11+) happy
-        # https://github.com/ARM-software/arm-trusted-firmware/blob/4ec2948fe3f65dba2f19e691e702f7de2949179c/make_helpers/toolchains/rk3399-m0.mk#L21-L22
-        rk3399-m0-oc = "${pkgsCross.arm-embedded.stdenv.cc.targetPrefix}objcopy";
-
-        buildInputs = [ openssl ];
-
-        makeFlags = [
-          "HOSTCC=$(CC_FOR_BUILD)"
-          "M0_CROSS_COMPILE=${pkgsCross.arm-embedded.stdenv.cc.targetPrefix}"
-          "CROSS_COMPILE=${stdenv.cc.targetPrefix}"
-          # Make the new toolchain guessing (from 2.11+) happy
-          "CC=${stdenv.cc.targetPrefix}cc"
-          "LD=${stdenv.cc.targetPrefix}cc"
-          "AS=${stdenv.cc.targetPrefix}cc"
-          "OC=${stdenv.cc.targetPrefix}objcopy"
-          "OD=${stdenv.cc.targetPrefix}objdump"
-          # Passing OpenSSL path according to docs/design/trusted-board-boot-build.rst
-          "OPENSSL_DIR=${openssl}"
-        ]
-        ++ (lib.optional (platform != null) "PLAT=${platform}")
-        ++ extraMakeFlags;
-
-        installPhase = ''
-          runHook preInstall
-
-          mkdir -p ${installDir}
-          cp ${lib.concatStringsSep " " filesToInstall} ${installDir}
-
-          runHook postInstall
-        '';
-
-        hardeningDisable = [ "all" ];
-        dontStrip = true;
-
-        # breaks secondary CPU bringup on at least RK3588, maybe others
-        env.NIX_CFLAGS_COMPILE = "-fomit-frame-pointer";
-
-        meta =
-          with lib;
-          {
-            homepage = "https://github.com/ARM-software/arm-trusted-firmware";
-            description = "Reference implementation of secure world software for ARMv8-A";
-            license = [
-              licenses.bsd3
-            ]
-            ++ lib.optionals (!deleteHDCPBlobBeforeBuild) [ licenses.unfreeRedistributable ];
-            maintainers = with maintainers; [ lopsided98 ];
-          }
-          // extraMeta;
-      }
-      // builtins.removeAttrs args [ "extraMeta" ]
-    )
-  );
-
-in
 {
-  inherit buildArmTrustedFirmware;
-
   armTrustedFirmwareTools = buildArmTrustedFirmware {
-    # Normally, arm-trusted-firmware builds the build tools for buildPlatform
-    # using CC_FOR_BUILD (or as it calls it HOSTCC). Since want to build them
-    # for the hostPlatform here, we trick it by overriding the HOSTCC setting
-    # and, to be safe, remove CC_FOR_BUILD from the environment.
-    depsBuildBuild = [ ];
```

Comment:
For the record, I don't understand what about this PR allows this to be removed. Out of curiosity, I did check out this PR and confirmed that adding/removing this line has no effect on the value of `nix eval .#armTrustedFirmwareTools`, so it seems to work.